data/hai/hai reveal 0 prologue.txt

```diff
@@ -0,0 +1,734 @@
+			`	"Captain <last>, is it? I don't believe we've met before," he says while shaking your hand, perhaps a little too firmly. "I'm Samuel Remington, head of the Syndicate's PR department. You may have not heard of me before, but I'm sure you've heard of my work. I..." He stops himself. "You know what, nevermind. We're not here to make small talk. We have bigger things to worry about. I'm looking forward to working alongside you to handle this emerging crisis."`
+				goto terry
+			label met
+			action
+				log "People" "Samuel Remington" `Remington is being sent as the representative of the Syndicate to help deal with the situation regarding the Hai.`
+			`	"Captain <last>, is it? It's been a good while since we last met," he says while shaking your hand, perhaps a little too firmly. "We met back on Hephaestus near the start of the war. It's been a while, so I'll forgive you if you've forgotten me. I'm looking forward to working alongside you to handle this emerging crisis."`
+			label terry
+			`	You note that his appearance is rather casual compared to what you would expect of someone of importance in the Syndicate, wearing khakis and a collared tee as opposed to the typical suit and tie business attire that you see on Syndicate businessmen. It is certainly in contrast to a young woman standing behind him, who is wearing the usual business attire.`
+			`	"Now, allow me to introduce you to Terry," Remington says while gesturing to the woman behind him.`
+			`	"Terry Adrianopoulos. Pleasure to meet you," she says with a broad smile and a slight bow. The young woman goes from calculating to gregarious in the blink of an eye as your attention shifts to her. She seems at once pleasant and approachable while at the same time being slightly off-putting and uncanny. It's an eerie feeling.`
+			`	"Terry is... well, let's call her my secretary," Remington says with a sly smile. Terry seems none too pleased to hear that, but she doesn't correct him.`
+			choice
+				`	"Alondo said you needed my help with something."`
+					goto help
+				`	"The Syndicate sent their PR team to deal with this?"`
+			`	"I'll have you know I'm considered something of a miracle worker among the Syndicate," Remington says in a sarcastic tone. "The Syndicate was sad to rent me out, but there's also an internal political situation going on within the Syndicate that has been causing me quite the headache recently." His eyes momentarily flick to Terry as he says that, but Remington presses on immediately. "It's nothing I can discuss in detail, with all the NDAs involved, you know, but something I'll certainly enjoy taking a break from."`
+			`	"Alondo said you needed my help," you say.`
+			label help
+			`	"Ah, yes. Well you see, I'd like to travel to <stopovers> to get the lay of the land around the conflict between the two Hai groups. Care to give me a ride there? It'd be just the two of us."`
+			`	"I will be accompanying you," Sayari says.`
```

Comment:
This would actually be an excellent point to flag the existence of Hiyamaharu.
I've been considering him to be the senior member of the Hai diplomatic core in specifically the Syndicate Relations division. Remington would likely have met/worked with him before and be a little surprised that Sayari was personally overseeing his business here.

Reply:
I'll keep this in mind, but I'd like to save that for a future PR, likely one which also removes Sayari from the Wanderer campaign so that she's not in two places at once.